.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,60 @@
+name: Feature Request
+description: Have an idea for an improvement to Scuffle? Your ideas are always welcome here
+title: "[Feature]: Scuffle needs [...]"
+labels: ["feature request"]
+assignees:
+  - ScuffleTV/reviewers
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### **Thank you for taking the time to fill out this Feature Request!**
+        Please provide as much information as possible. To make sure we have all the details we need to get started, please fill out the form below.
+  - type: markdown
+    attributes:
+      value: ---
+  - type: textarea
+    id: idea
+    attributes:
+      label: Tell us about your Wish
+      description: |
+        Describe the behavior you are looking for, pitch an idea, or make a suggestion for improvement.
+    validations:
+      required: true
+  - type: input
+    id: bugrelation
+    attributes:
+      label: Is your feature related to a bug
+      description: |
+        Refer to a an existing bug, you can use `#bugid`
+    validations:
+      required: false
+  - type: textarea
+    id: alternatives
+    attributes:
+      label: Alternatives
+      description: |
+        Have you thought about alternative solutions or ways of implementation? Do you have an idea for how this could be a reality? Let us know!
+  - type: textarea
+    id: info
+    attributes:
+      label: Additional Info
+      description: |
+        Provide additional information or links to resources that may be of assistance in the creation process.
+    validations:
+      required: false
+  - type: checkboxes
+    id: participation
+    attributes:
+      label: Participation
+      options:
+        - label: I am willing to submit a pull request to implement this
+          required: false
+  - type: checkboxes
+    id: disclaimer
+    attributes:
+      label: Disclaimer
+      description: I am confident that this is the first feature request to my Wish
```

Comment:
Any specific reasons to actually have this?

Reply:
To prevent duplicate feature request

Reply:
Is this supposed to mean "I made sure there have been no duplicate suggestions"

Reply:
Means the same thing - just rewritten a little to make it seem "looser". like e.g. discord-style

Reply:
Alright, just change it to "I have checked that this has not been suggested before"

Reply:
i agree, this sounds better

Reply:
fixed